api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -163,47 +174,84 @@ public long getQueryBatchSize() {
     return jdbcTemplate.query(
         String.format(
             "SELECT \n"
-                + "  u.about_you,\n"
-                + "  u.area_of_research,\n"
-                + "  u.compliance_training_bypass_time,\n"
-                + "  u.compliance_training_completion_time,\n"
-                + "  u.compliance_training_expiration_time,\n"
-                + "  u.contact_email,\n"
-                + "  u.creation_time,\n"
-                + "  u.current_position,\n"
-                + "  u.data_access_level,\n"
-                + "  u.data_use_agreement_bypass_time,\n"
-                + "  u.data_use_agreement_completion_time,\n"
-                + "  u.data_use_agreement_signed_version,\n"
-                + "  u.demographic_survey_completion_time,\n"
-                + "  u.disabled,\n"
-                + "  u.era_commons_bypass_time,\n"
-                + "  u.era_commons_completion_time,\n"
-                + "  u.family_name,\n"
-                + "  u.first_registration_completion_time,\n"
-                + "  u.first_sign_in_time,\n"
-                + "  u.free_tier_credits_limit_days_override,\n"
-                + "  u.free_tier_credits_limit_dollars_override,\n"
-                + "  u.given_name,\n"
-                + "  u.last_modified_time,\n"
-                + "  u.professional_url,\n"
-                + "  u.two_factor_auth_bypass_time,\n"
-                + "  u.two_factor_auth_completion_time,\n"
                 + "  u.user_id,\n"
+                + "  u.about_you AS about_you,\n"
```

Comment:
Why is `AS` needed now, where it wasn't before?

Reply:
Ah, that is optional field, if AS is not present, it will use the original column name(if there is no conflict, e.g., both user and demographic_survey gives you user_id). Seems they all match to the original column name, and no duplicate, let me remove them.

Reply:
(it was addded because the column name became `GROUP_CONCAT(about_you)`)